connectivity/drivers/lora/TARGET_STM32WL/STM32WL_radio_driver.cpp

```diff
@@ -16,8 +16,49 @@
 #include "STM32WL_radio_driver.h"
 #include "drivers/DigitalOut.h"
 
-
 /* Sets up radio switch position according to the radio mode */
+#if defined (TARGET_LORA_E5) || defined (TARGET_RAK3172)
```

Comment:
I think this list of define will not be "nice" when we will add again more targets? :-)

Proposition:
- create a new sub-directory here TARGET_NUCLEO_WL55JC with thi scurrent file inside
- you can the create TARGET_LORA_E5 and TARGET_RAK3172 with the specific file content
?

Reply:
Yeah don't like these define either.  Subfolder is great, the deal is that I'm not a guru of cmake, so not sure how to tell which dir to include depending on target, I'll try that.

Reply:
We shouldn't have targets configuration in libraries - these should be hw agnostic.
Instead, you can propose a sw configuration to enable/disable features. This could be used by any target configuration (e.g mbed_app.json).

Reply:
done, look likes it's was easier than I thought